tests/e2e/run_hosted_claimed_e2e.py

```
@@ -289,29 +470,111 @@ def assert_gecko(profile: Path) -> None:
+
+def esent_copy_command(source: Path, destination: Path) -> list[str]:
+    return ["esentutl.exe", "/y", str(source), f"/d{destination}", "/o"]
+
+
+def snapshot_internet_explorer_store(cookie_file: Path) -> Path:
+    """Copy the real ESE store out of the Windows WebCache share lock."""
+
+    destination = (
+        Path(tempfile.gettempdir()) / f"rookie-ie-WebCacheV01-{time.time_ns()}.dat"
+    )
+    completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)